### PR TITLE
General Page Layout

### PR DESCRIPTION
**Pull request by jubalm**
This will be the layout structure for Assets page with a demo on `#layout` route. Notably need a review on other devices than mac and should not have the scrollbar artifacts previously noticed on assets listing.

[UPDATE]: remove demo pages

### CODE REVIEW

**Comment by 0xjimmy** [comment]
Use `overflow-auto` instead if `overflow-scroll`, as chromium renders empty scroll bars even if the content is too small to scroll.

File: app/ts/store/tokens.ts [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+	chainId: number
+	name: string
+	address: string
+	symbol: string
+	decimals: number
+}
+
+export const DEFAULT_TOKENS: TokenMeta[] = [
+	{
+		chainId: 1,
```
Comment (KillariDev) [inline app/ts/store/tokens.ts:96]: image is missing from here?

**Review by KillariDev — APPROVED** [review]
This adds token images again like the previous PR?

File: app/ts/store/tokens.ts (hunk 1 above)
Comment (jubalm) [reply app/ts/store/tokens.ts]: image icon will be loaded as publicPath + lowecased contract address + file extension else will show default ethereum icon

**Comment by jubalm** [comment]
> This adds token images again like the previous PR?

rebased to remove confusion

**Comment by MicahZoltu** [comment]
This PR just removed the demo pages and nothing, why was that?  I found the demo pages to be very valuable.

**Pull request merged**